src/flisp/Makefile

```diff
@@ -59,9 +62,9 @@ $(BUILDDIR)/flisp.dbg.obj:  flisp.c cvalues.c types.c flisp.h print.c read.c equ
 $(BUILDDIR)/flmain.o:  flmain.c flisp.h
 $(BUILDDIR)/flmain.dbg.obj: flmain.c flisp.h
 
-$(LLT_release): $(LLTDIR)/*.h $(LLTDIR)/*.c
+$(LLT_release): $(LLTDIR)/*.h $(LLTDIR)/*.c flisp-deps
```

Comment:
It's invalid to depend on a `.PHONY` target

Reply:
I'm not sure I would call it _invalid_, but it _does_ mean that `$(LLT_release)` will always be out of date, which isn't what I intended.